Content.Server/Stack/StackSystem.cs

```diff
@@ -91,7 +94,7 @@ private void OnStackAlternativeInteract(EntityUid uid, StackComponent stack, Get
             {
                 Text = Loc.GetString("comp-stack-split-halve"),
                 Category = VerbCategory.Split,
-                Act = () => UserSplit(uid, args.User, stack.Count / 2, stack),
+                Act = () => UserSplit(uid , args.User , stack.Count/2, stack),
```

Comment:
do not make random formatting changes in files you're only just touching. especially because these formatting changes make it so that the code no longer follows our style conventions, revert this.

Reply:
ok